spec/models/report_template_spec.rb

```diff
@@ -69,4 +69,40 @@
       expect(translation_test_report_template.name).to include('from=>en to=>pl text=>Test report template name')
     end
   end
+
+  describe 'clone' do
+    subject { report_template.clone }
+
+    let!(:report_template) { create(:report_template, :with_logo, :with_sections) }
+
+    it 'create a copy in the exising session' do
```

Comment:
Maybe checking if the cloned report is in the same session as the original one

Reply:
If you raise this I think it is